public/app/leaderboard.tag

```diff
@@ -30,6 +126,25 @@
       <td class="accounting">{ stat.health_given }</td>
     </tr>
   </table>
+  <div class="leaderboard">
+    <div each={ stat, i in stats } class="leaderboard__hero  hero-stats  { user.getCurrentUserClass(stat.github_login) }">
+      <div class="hero-stats__item  hero-stats__attribute  hero-stats__rank">Rank</div>
```

Comment:
I'm not sure we need the `hero-stats__item` class here because the CSS selector could change from `.hero-stats__item` to `.hero-stats > div` and accomplish the same thing without needing all that extra markup.